paasta_tools/marathon_serviceinit.py

```diff
@@ -108,10 +108,23 @@ def status_marathon_job(service, instance, app_id, normal_instance_count, client
     if marathon_tools.is_app_id_running(app_id, client):
         app = client.get_app(app_id)
         running_instances = app.tasks_running
+
         if len(app.deployments) == 0:
             deploy_status = PaastaColors.bold("Running")
+        elif app.instances == 0 and app.tasks_running == 0:
+            deploy_status = PaastaColors.grey("Stopped")
         else:
-            deploy_status = PaastaColors.yellow("Deploying")
+            # App is currently deploying so we should check the launch queue for more info
+            is_overdue, backoff_seconds = marathon_tools.get_app_queue_status(client, app_id)
+
+            if is_overdue:
+                deploy_status = PaastaColors.red("Waiting for resources")
+            elif backoff_seconds:
+                delay_time = PaastaColors.grey("%s seconds" % backoff_seconds)
+                deploy_status = "%s (%s)" % (PaastaColors.red("Delayed"), delay_time)
```

Comment:
I would like to make this more obvious to users what Waiting and Delayed means, but I can't think of good signals.

Any ideas on making this more verbose?

Maybe:

   "Waiting: new tasks are not launching due to lack of capacity"

and

   "Delayed: Next task won't launch for %s seconds due to previous failures"

?